pkg/config/nerdctl_config_applier.go

```diff
@@ -91,8 +91,8 @@ func addLineToBashrc(fs afero.Fs, profileFilePath string, profStr string, cmd st
 func updateEnvironment(fs afero.Fs, fc *Finch, finchDir, homeDir, limaVMHomeDir string) error {
 	cmdArr := []string{
 		`export DOCKER_CONFIG="$FINCH_DIR"`,
-		"[ -L /usr/local/bin/docker-credential-ecr-login ] " +
-			`|| sudo ln -s "$FINCH_DIR"/cred-helpers/docker-credential-ecr-login /usr/local/bin/`,
+		`for f in "$FINCH_DIR"/cred-helpers/docker-credential-*; do ` +
```

Comment:
Do we want to validate the files before copying? I am thinking if it will be better to verify the files with the cred_helpers config value. This way it makes the configuration clear and we can also throw an error here if the particular cred-helper binary is not found. wdyt?

Reply:
Discussed offline. Instead of linking all files from credential helpers matching pattern; will iterate over all credential helpers in configuration and only link those files to the path. The result is any configuration which lists a binary that is not in cred-helpers directory will fail.